### PR TITLE
Add validation for bounding boxes

### PR DESCRIPTION
**Pull request by moradology**
This PR aims to provide some sanity checks on bounding boxes with helpful error messages in case they are malformed in either min/max ordering or in terms of the number of values (4 or 6 being absolutely required)

### CODE REVIEW

File: geojson_pydantic/features.py [hunk 1]
```diff
@@ -27,6 +27,19 @@ class Config:
...
+            if bbox[0] > bbox[3] or bbox[1] > bbox[4] or bbox[2] > bbox[5]:
+                raise ValueError("BBox must be in the form [west, south, bottom, east, north, top]")
+        elif len(bbox) == 4:
+            if bbox[0] > bbox[2] or bbox[1] > bbox[3]:
+                raise ValueError("BBox must be in the form [west, south, east, north]")
+        else:
+            raise ValueError("BBox must be in the form [west, south, east, north] or [west, south, bottom, east, north, top]")
+        return bbox 
+
+
```
Comment (vincentsarago) [inline geojson_pydantic/features.py:42]: would you be able to add some tests for this 🙏

Comment (moradology) [reply geojson_pydantic/features.py:42]: That was fast! Yes, absolutely

Comment (moradology) [reply geojson_pydantic/features.py:46]: The tests are pretty simple, but I think they get the job done. Let me know if I can do anything else!

**Review by vincentsarago — APPROVED** [review]

**Comment by vincentsarago** [comment]
@moradology that's said, this will only the `bbox` attribute of a geometry, not the geometry coordinates itself.

**Comment by moradology** [comment]
Would it be desirable for me to add some geometry validation as well? If so, I wonder if you have some ideas about what features of vectors would be worth checking (and not be likely to add undesirable performance penalties)

**Review by geospatial-jeff — APPROVED** [review]
LGTM

**Pull request merged**